src/Model/ClearableManagerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Doctrine\Model;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+interface ClearableManagerInterface
+{
+    /**
+     * Clears the object manager for the given model.
```

Comment:
Checking the implementation, I wonder what happens if the argument value is `null`? The whole manager is cleared? If that is the case, I think we should make the behavior clear in this description.

Reply:
This null behavior is copied from the original doctrine method: https://github.com/doctrine/persistence/blob/2.2.x/lib/Doctrine/Persistence/ObjectManager.php#L69-L77

Reply:
IMO, we could use a similar description for the method and the argument then.